packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -454,7 +470,33 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 												</div>
 											))}
 										</div>
-									</>
+									</div>
+								) : null}
+
+								{history.length && !hideHistory ? (
+									<div className="ss__autocomplete__terms__history">
```

Comment:
May want to not render the title divs if someone provides `<Autocomplete historyTitle={''}>`
Terms and Trending has this:
```
{trendingTitle ? (
    <div className="ss__autocomplete__title ss__autocomplete__title--trending">
        <h5>{trendingTitle}</h5>
    </div>
) : null}
```